discovery/triton/triton.go

```diff
@@ -219,6 +227,13 @@ func (d *Discovery) refresh() (tg *targetgroup.Group, err error) {
 		}
 		addr := fmt.Sprintf("%s.%s:%d", container.VMUUID, d.sdConfig.DNSSuffix, d.sdConfig.Port)
 		labels[model.AddressLabel] = model.LabelValue(addr)
+
+		if container.Groups != nil {
+			for _, v := range container.Groups {
+				labels[tritonLabelGroups+"_"+model.LabelName(v)] = model.LabelValue(v)
```

Comment:
The way we handle arrays is to have a single label where they're joined with commas, and also surrounded with commas.

Reply:
done